Gravity-Notes/src/Utils/DeleteFromArchive.js

```diff
@@ -1,13 +1,14 @@
 import axios from "axios"
+import toast from "react-hot-toast"
 
 export const deleteFromArchive = async (note,authState,noteDispatch)=>{
     try {
         const res = await axios.delete(`/api/archives/delete/${note._id}`,{headers:{authorization: authState.token}})
```

Comment:
in url why not use only _id ? you can directly pass id instead of passing note object in parameter.